File: vendaval.client/src/app/product/product.ts
import { BaseModel } from "../shared/common/interfaces/BaseModel";
import { ProductType } from "./product-type";

export interface Product extends BaseModel{
  name: string;
  description: string;
  price: number;
  stock: number;
  image: string;
  category: ProductType;
  avaliation: number
}

File: Vendaval.Domain/Entities/Product.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Vendaval.Domain.Enums;

namespace Vendaval.Domain.Entities
{
    public class Product : BaseModel
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        public int Stock { get; set; }
        public string Image { get; set; }
        public ProductType Category { get; set; }
        public float Avaliation { get; set; }
    }
}
